avocado/plugins/spawners/podman.py

```diff
@@ -29,14 +29,35 @@ def initialize(self):
             section=section,
             key='image',
             help_msg=help_msg,
-            default='fedora:31')
+            # quay.io/willianr/avocado-82lts
+            default='avocado-82lts')
```

Comment:
You could have used the entire URL here, podman will accept it happily.  But anyway, changing this is annoying, so I sent: https://github.com/avocado-framework/avocado/pull/4284

Reply:
Ah, don't worry about looking at changes here, it was just to make the test easy. As I had a local 82lts image, I forgot to change that before submitting the PR. Thanks for sending #4284, I had it on my to-do list, but never got to it!